### PR TITLE
update the readme and change log: support local JRE key store

### PR DESCRIPTION
**Pull request by michaelqi793**

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## 1.0.0-beta.7 (Unreleased)
 
+### New Features
+ - Support the local JRE key store
```
Comment (yiliuTo) [inline sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md:6]: - Please add the related PR link here.
- add the changelog of starter as well.

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md [hunk 2]
```diff
@@ -278,6 +278,20 @@ private static class ClientPrivateKeyStrategy implements PrivateKeyStrategy {
 }
 ```
 
+
+### Enable local JRE key store
+
+The local JRE key store can also work with Key Vault key store. To enable this, add the following configuration:
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md:284]: I think we could give more clear description that it indeed is used as a trust store. Please introduce about the context as well, for example, which is used by default, and what will happen when the below property is enabled. Thanks.

**Review by yiliuTo — CHANGES_REQUESTED** [review]

File: sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 ## 1.0.0-beta.7 (Unreleased)
 
+### New Features
+ - Support the local JRE key store. Please refer to [Support JRE Key Store][PR_Support_JRE_KeyStore]
```
Comment (chenrujun) [inline sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md:6]:
```
Support the local JRE key store. ([#20548])
```

File: sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md [hunk 4]
```diff
@@ -30,3 +33,8 @@
 
 ## 1.0.0-beta.1 (2020-10-21)
  - First release.
+
+
+
+<!-- LINKS -->
+[PR_Support_JRE_KeyStore]: https://github.com/Azure/azure-sdk-for-java/pull/20548
```
Comment (chenrujun) [inline sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md:40]:
```
[#20548](https://github.com/Azure/azure-sdk-for-java/pull/20548)
```

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,7 @@
 ### New Features
 - Upgrade to [spring-boot-dependencies:2.4.5](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.4.5/spring-boot-dependencies-2.4.5.pom).
 - Upgrade to [spring-cloud-dependencies:2020.0.2](https://repo.maven.apache.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2020.0.2/spring-cloud-dependencies-2020.0.2.pom).
+- Support the local JRE key store. Please refer to [Support JRE Key Store][PR_Support_JRE_KeyStore]
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md:7]: Same here.

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md [hunk 6]
```diff
@@ -25,3 +26,7 @@
 
 ## 3.0.0-beta.1 (2020-10-21)
  - First release.
+
+
+<!-- LINKS -->
+[PR_Support_JRE_KeyStore]: https://github.com/Azure/azure-sdk-for-java/pull/20548
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md:32]: Here.

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md [hunk 7]
```diff
@@ -278,6 +278,20 @@ private static class ClientPrivateKeyStrategy implements PrivateKeyStrategy {
 }
 ```
 
+
+### Enable local JRE key store
+
+If you use key vault key store but don't override the trust manager factory, by default, the JRE trust manager will be used and will employ only the key vault key store. In this case the local JRE key store will not work. To let it also work with Key Vault key store, you need use the key vault trust manager which employs both the key vault key store and the local JRE keystore. To do this, add the following configuration:
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md:284]: We need to discuss about this.

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md (hunk 7 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md:284]: Can we put all certs in JRE key store into KeyvaultKeyStore? So we can support the certs in local JRE key store without override trust manager factory.

Comment (michaelqi793) [reply sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md:284]: > Can we put all certs in JRE key store into KeyvaultKeyStore? So we can support the certs in local JRE key store without override trust manager factory.

no need in this way, if you want to implement this in  key store level, just let key vault key store hold the jre key store as an assistance delegate.

**Comment by chenrujun** [comment]
@michaelqi793  is developing this feature by another way, so closing this PR.

**Pull request closed without merging**